src/Swarm/TUI/Model/StateUpdate.hs

```diff
@@ -119,6 +118,7 @@ startGameWithSeed userSeed siPair@(_scene, si) toRun = do
     . scenarioStatus
     .= Played (Metric Attempted $ ProgressStats t emptyAttemptMetric) (prevBest t)
   scenarioToAppState siPair userSeed toRun
+  gameState . currentScenarioPath .= Just p
```

Comment:
@kostmo could you add a short comment here please? I would hate to break this again in refactoring. 😅